### PR TITLE
Data si timpul

### PR DESCRIPTION
**Pull request by ioanachi**

### CODE REVIEW

File: hello.php [hunk 1]
```diff
@@ -50,10 +50,13 @@ function hello_dolly_get_lyric() {
 	return wptexturize( $lyrics[ mt_rand( 0, count( $lyrics ) - 1 ) ] );
 }
 
+
 // This just echoes the chosen line, we'll position it later
 function hello_dolly() {
+	$timpul = time();
+	$data = date("Y/m/d/H.a");
```
Comment (turcuciprian) [inline hello.php]: formatul sa fie 2017/08/08 17:23:00

**Review by turcuciprian — CHANGES_REQUESTED** [review]

**Review by turcuciprian — APPROVED** [review]

**Review by turcuciprian — APPROVED** [review]
I can't merge anything!!!

**Comment by ioanachi** [comment]
multe mai vrei!
why can't you merge anything?

**Comment by turcuciprian** [comment]
:))

**Comment by turcuciprian** [comment]
I don't have the merge button. Change my permission please. In contributors

**Pull request merged**